pages/src/source/tutorials/datapacks.md

```diff
@@ -157,12 +157,12 @@ The <code>"description"</code>'s value is displayed to the user as an in-game ti
 | <code>minecraft</code>       | <code>small_flowers</code>             | Grown by a building level 1 or 2 {% worker_link florist %}, if they have a valid block form, and are in #minecolonies:florist_flowers. |
 | <code>minecraft</code>       | <code>wool</code>                      | Used to by the Dyer to produce white wool, if not already White Wool. |
 
-**Some Vanilla and Forge Item Tags are very expansive, or are used by some mods in ways that might surprise you. See [list of tags](https://Minecraft.fandom.com/wiki/Tag#List_of_tags) for Minecraft behaviors.**
+**Some Vanilla and Forge Item Tags are very expansive, or are used by some mods in ways that might surprise you. See [list of tags](https://Minecraft.wiki/w/Tag #List_of_tags) for Minecraft behaviors.**
```

Comment:
There should not be a space before URL hashes (`Tag #List_of_tags` -> `Tag#List_of_tags`)